scalafix-tests/src/test/resources/checkSyntax/RemoveXmlLiterals.source

```diff
@@ -0,0 +1,112 @@
+rewrites = [RemoveXmlLiterals]
+
+<<< don't add import when not needed
+object A {}
+>>>
+object A {}
+
+<<< single line, no splice
+object A {
+  <div></div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  xml"<div></div>"
+}
+
+<<< single line, triple quoted
+object A {
+  val a = <div b="Hello"/>
+  val b = <a>\</a>
+}
+>>>
+import scala.xml.quote._
+object A {
+  val a = xml"""<div b="Hello"/>"""
+  val b = xml"""<a>\</a>"""
```

Comment:
How come this is triple quote while the test above is single quote? Is it the escape `\`?